app/views/component-listing.html

```diff
@@ -40,7 +40,7 @@
 			<tbody>
 			{% for group in components %}
 				{% if group.modules is not empty %}
-					<tr class="component-group">
+					<tr class="component-group searchable">
```

Comment:
Can you make this `js-searchable` (in origami components it would be `o-js-` but since this isnt an origami component...